src/main.rs

```diff
@@ -39,12 +41,23 @@ async fn main() -> Result<()> {
     let settings = Settings::new()?;
     debug!("Settings: {:?}", settings);
 
-    match cli_args.action {
-        Action::Config(cli_args) => actions::config::main(settings, cli_args).await,
-        Action::Install => actions::install::main(settings).await,
-        Action::Uninstall => actions::uninstall::main(settings).await,
-        Action::PrepareCommitMsg(cli_args) => {
-            actions::prepare_commit_msg::main(settings, cli_args).await
+    if let Some(subcommand) = cli_args.action {
```

Comment:
Rather than nesting `match` inside the `if let`, use one single `match` statement for all these cases